src/components/VmDetails/cards/NicsCard/NicEditor.js

```diff
@@ -190,7 +190,11 @@ class NicEditor extends Component {
         </Modal.Header>
         <Modal.Body>
 
-          <Form horizontal>
+          <Form
+            horizontal
+            onSubmit={e => { e.preventDefault() }}
+            id={`${modalId}-form`}
```

Comment:
For keeping consistency here must be `id={${modalId}-modal-form}` too

Reply:
https://github.com/oVirt/ovirt-web-ui/blob/b28c641e7bb6c5baa98935baa97f0f6e8109a65b/src/components/VmDetails/cards/NicsCard/NicEditor.js#L152-L154

It already is `props.id + '-modal' + '-form'`

Reply:
Oh, ok, I missed it.